examples/gateway-hooks/authorized-subgraph/src/schema.rs

```diff
@@ -0,0 +1,160 @@
+use std::collections::HashMap;
+
+use async_graphql::{Context, Object, TypeDirective};
+
+#[TypeDirective(
+    name = "authorized",
+    location = "FieldDefinition",
+    location = "Object",
+    composable = "https://custom.spec.dev/extension/v1.0"
+)]
+fn authorized(rule: String, arguments: Option<String>) {}
```

Comment:
rule can be removed, could add `metadata`, but not sure of the type that should be used with async-graphql, with `serde_json::Value` it just serializes it to a string... maybe `async_graphql::Value`, but haven't tried it

Reply:
And you're never using it in the subgraph?